src/config/dynamic-theme-fixes.config

```diff
@@ -2072,6 +2072,25 @@ td.c {
 
 ================================
 
+localhost:88*
+
+INVERT
+#ipython_notebook
+
+CSS
+.output_area img, .text_cell_render img {
+    background-color: ${black} !important;
+}
+.CodeMirror-selected {
```

Comment:
We can move this to * as CodeMirror is widely used for different task.

Reply:
Done.